File: backend/scripts/test_db.js
const { PrismaClient } = require('@prisma/client');

async function test() {
    // Password: !WOwFsnVm7q:[PGn4IR2qoI5i|*r
    // Encoded: %21WOwFsnVm7q%3A%5BPGn4IR2qoI5i%7C%2Ar
    const url = "postgresql://postgres:%21WOwFsnVm7q%3A%5BPGn4IR2qoI5i%7C%2Ar@poker.c548oskckol3.sa-east-1.rds.amazonaws.com:5432/postgres?sslmode=require";
    
    process.env.DATABASE_URL = url;
    console.log("Testing new credentials from Secrets Manager...");
    const prisma = new PrismaClient();
    try {
        await prisma.$connect();
        console.log("✅ PERFECT! THE PASSWORD WORKS!");
        // Check if saas_crm database exists or use postgres for now
        await prisma.$disconnect();
        process.exit(0);
    } catch (e) {
        console.log(`❌ STILL FAILING: ${e.message}`);
        process.exit(1);
    }
}

test();

File: backend/scripts/test_db.py
import asyncio
from prisma import Prisma

async def test():
    urls = [
        "postgresql://postgres:Mh4M9k%21HPVq%2ED2cNG%24W%3Fn4%3EF1%2E%3FQ@poker.c548oskckol3.sa-east-1.rds.amazonaws.com:5432/postgres?sslmode=require",
        "postgresql://postgres:Wzwz%402025Ciberntico@44.192.122.113:5432/postgres?sslmode=require",
        "postgresql://postgres:admin123@poker.c548oskckol3.sa-east-1.rds.amazonaws.com:5432/postgres?sslmode=require"
    ]
    
    for url in urls:
        print(f"Testing {url.split('@')[1]}...")
        p = Prisma(datasource={"url": url})
        try:
            await p.connect()
            print(f"✅ SUCCESS with {url.split('@')[1]}")
            await p.disconnect()
            break
        except Exception as e:
            print(f"❌ FAILED: {str(e)[:100]}")

if __name__ == "__main__":
    asyncio.run(test())
